spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -100,6 +100,18 @@
     end
   end
 
+  describe "#unpublish" do
+    it "sets status to draft" do
+      stub_publishing_api
+
+      post :unpublish, params: { step_by_step_page_id: step_by_step_page.id, redirect_url: "/somewhere" }
+
+      expect(step_by_step_page.status).to eq "draft"
```

Comment:
I think the `step_by_step_page` here is already in draft before we `unpublish` it. Could you set `step_by_step_page` to a published one?